hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/mor/MergeOnReadInputSplit.java

```diff
@@ -67,6 +67,34 @@ public MergeOnReadInputSplit(
     this.instantRange = Option.ofNullable(instantRange);
   }
 
+  public MergeOnReadInputSplit(
+          int splitNum,
+          @Nullable String basePath,
+          Option<List<String>> logPaths,
+          String latestCommit,
+          String tablePath,
+          long maxCompactionMemoryInBytes,
+          String mergeType,
+          @Nullable InstantRange instantRange,String fileId ) {
```

Comment:
Why we must add a new constructor ?

Reply:
I don't have to add it. Let me change it，thank you !~